registration/templates/registration/failed_email.txt

```diff
@@ -0,0 +1,9 @@
+{% autoescape off %}
+Hi!
+
+The instance for {{ application.user.name }} failed to provision.
+
+Happy trails!
```

Comment:
??
I have not seen this sign-off before is it intentional?

Reply:
This is a test template that was committed by mistake, sorry.